brainkeeper/src/app/classes/storable-person.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Person } from './person';
+import { StorablePerson } from './storable-person';
+
+describe('StorablePerson', () => {
+    describe('fromPerson', () => {
+        it('converts Person to StorablePerson when id is null', () => {
+            const person = new Person('Jon Doe', 'abc');
+            expect(person.id).toBeNull();
+            const storablePerson = StorablePerson.fromPerson(person);
+            expect(storablePerson.id).toBeUndefined();
+            expect(storablePerson.name).toBe('Jon Doe');
+            expect(storablePerson.picture).toBe('abc');
+            expect(storablePerson.score).toBe(0);
+        });
+
+        it('converts Person to StorablePerson when id is 0', () => {
+            const person = new Person('Jon Doe', 'abc', 0, 1);
+            const storablePerson = StorablePerson.fromPerson(person);
+            expect(storablePerson.id).toBe(0);
+            expect(storablePerson.name).toBe('Jon Doe');
+            expect(storablePerson.picture).toBe('abc');
+            expect(storablePerson.score).toBe(1);
+        });
+    });
+
+    describe('toPerson', () => {
+        it('converts Person to StorablePerson when id is undefined', () => {
```

Comment:
Shouldn't it say 'convert StorablePerson to Person with no id'?

Reply:
Fixed with ef46d81